index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <style> @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@300&display=swap'); </style>
+    <style> @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@300&display=swap'); </style>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="mobile-view">
+            <header class="mobile-header">
+                <div class="display-header">
+                    <p id="time">9:41</p>
+                    <p id="portfolio-name">John Doe </p>
+                </div>
+                <div class="signal-div">
+                    <div class="signal-div-items">
+                        <img src="images/Cellular Connection.png" alt="cellular connection image">
+                        <img src="images/Wifi.png" alt="wifi image">
+                        <img src="images/Battery.png" alt="Batteryimage">
+                    </div>
+                    <div class="union-div">
+                        <img src="images/Union.png" alt="hamburger">
+                    </div>
+                </div>
+            </header>
+            <div class="headline-section">
+                <div class="name-div">
+                    <p class="name">Here There.</p>
+                    <p class="name">I am Alex</p>
+                </div>
+                <div class="job-title-div">
+                    <p id="job-title">I am a Software Developer</p>
+                </div>
+                <div>
+                    <p id="introduction">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
+                </div>
+                <div class="social">
+                    <img class="icon-interactions" src="images/github.png" alt="github-image">
+                    <img class="icon-interactions" src="images/Group.png" alt="union-icon">
+                    <img class="icon-interactions" src="images/hand.png" alt="hand">
+                    <img class="icon-interactions" src="images/twitter.png" alt="twitter bird">
+                    <img class="icon-interactions" src="images/Vector.png" alt="M-sign">
+                </div>
```

Comment:
- Let's use the ul tag to wrap the images as this is the correct semantic tag needed for the list of images like this
   
   ```
      <ul class="social">
            <li><a href=""><img src="" alt=""></a></li>
            <li><a href=""><img src="" alt=""></a></li>
            <li><a href=""><img src="" alt=""></a></li>
            <li><a href=""><img src="" alt=""></a></li>
            <li><a href=""><img src="" alt=""></a></li>
       </ul>

```